.travis/travis-functions.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/env bash
+#
+# Copyright 2019 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+function updateVersionInRenku() {
+  echo "Updating renku-gateway version to $CHART_VERSION in Renku's requirements.yaml"
+
+  # cloning the renku repo
+  cd || exit
+  git clone https://github.com/SwissDataScienceCenter/renku.git
+  cd renku || exit
+
+  # changing the version
+  sed -i "/- name: renku-gateway/{n;n;n;s/.*/  version: $CHART_VERSION/}" charts/renku/requirements.yaml
```

Comment:
in the future we should use `yq` to do this in a more human-readable way maybe

Reply:
Agreed.